### DISCUSSION TITLE
Multiple monitors, mouse movement

### DISCUSSION THREAD

**Discussion by zabihexes** [any1/wayvnc#256] [Q&A]
Moving mouse is kinda scuffed on multiple monitors for me. The mouse is basically 2x times faster if I move horizontally.(I have 2 monitors next to each other.). The mouse can leave the monitor that I am vnced into.  I can still like click with it and use it(Assuming I can see my monitor physically to know where the cursor is.)
Is there anything I can do about it or is it a bug?

I am on hyprland on Fedora. It is awfull on some vnc clients that zoom in where the cursor is supposed to be.(Since it is actually elsewhere.)

**Comment by any1** [comment]
The compositor needs to bind the virtual pointer to the captured output as requested by `zwlr_virtual_pointer_manager_v1_create_virtual_pointer_with_output`. Perhaps this is not implemented in hyprland?

**Comment by svyatoclav** [comment]
Hyprland 35.0 on Artix with 2 monitors: 1920x1080 and 2560x1440. Same issue: mouse cursor(s) doubles (cursor of client and cursor of server) and not synchronized. I assume it is because of only "one" area of monitor(s), but it's actually two of them. Horizontal movement on server is about 2x times faster also because of that and the mouse pointer doesn't have proper position tho 
 
Is there any way to "share" only one monitor with proper mouse movement? I tried `wayvnc -oDP-2` and `wayvnc -oDP-1` but nothing changed: cursor still mismatch position. Or can I share even two at the same time? Any fixes yet? Thanks.
